routes/cube_routes.js

```diff
@@ -3056,6 +3056,53 @@ router.post(
   }),
 );
 
+router.post(
+  '/api/maybe/update/:id',
+  ensureAuth,
+  util.wrapAsyncApi(async (req, res) => {
+    const cube = await Cube.findOne(build_id_query(req.params.id));
+    if (!req.user._id.equals(cube.owner)) {
+      return res.status(403).send({
+        success: 'false',
+        message: 'Maybeboard can only be updated by cube owner.',
+      });
+    }
+
+    const card = cube.maybe.find((card) => card._id.equals(req.body.id));
+    if (!card) {
+      return res.status(404).send({
+        success: 'false',
+        message: 'No card found to update.',
+      });
+    }
+
+    const updated = req.body.updated;
+    if (!updated) {
+      return res.status(400).send({
+        success: 'false',
+        message: 'Bad request.',
+      });
+    }
+    const newVersion = updated.cardID && updated.cardID !== card.cardID;
+    for (field of ['cardID', 'status', 'finish', 'cmc', 'type_line', 'imgUrl', 'colors']) {
+      if (updated.hasOwnProperty(field)) {
+        card[field] = updated[field];
+      }
+    }
+    await cube.save();
+
+    if (newVersion) {
+      return res.status(200).send({
+        success: 'true',
+        details: carddb.cardFromId(card.cardID),
+      });
+    } else {
```

Comment:
else is not necessary 
(removing this removes a bit of nesting which is a plus)

Reply:
This is a stylistic point but I prefer else here... if you want me to change it i can

Reply:
Chiming in I prefer having the else as well. It's a little more nesting but it helps make the control flow explicit.

Reply:
To me, all of the 3 earlier branches that return follow the pattern of not using else, and if they used else it would cause a lot of chained nesting. This is the last branch so it looks like a different case but it's isn't actually. I'd prefer to be consistent avoiding unnecessary else - and make the code read as linear as possible

Reply:
I'll submit to your demands this time but next time.............

Reply:
Also, while acknowledging the fact that this is an extremely unimportant argument, the point of this style is to point out that there are two alternate paths which both constitute "success." Normally, the if(x) return; pattern indicates that you're exiting due to some abnormal condition, but these are both equally successful return statements.